serverless/cmd/gcp/README.md

```diff
@@ -0,0 +1,8 @@
+TODO(jayhou): make this info more friendly. Currently contains rough notes of
```

Comment:
Ah, it looks like the files are in both locations now in the PR!
I'll stop reviewing here for now :)

Reply:
I have removed the "internal" version now.